sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_create_test_live.cpp

```diff
@@ -211,5 +231,88 @@ TEST_F(KeyVaultKeyClient, CreateRsaHsmKey)
     CheckValidResponse(keyResponse);
     auto keyVaultKey = keyResponse.Value;
     EXPECT_EQ(keyVaultKey.Name(), keyName);
+    EXPECT_FALSE(keyResponse.Value.Properties.ReleasePolicy.HasValue());
+    EXPECT_TRUE(keyVaultKey.Properties.Enabled.Value());
+  }
+}
+
+std::string BinaryToHexString(std::vector<uint8_t> const& src)
+{
+  static constexpr char hexMap[]
+      = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'};
+  std::string output(static_cast<size_t>(src.size()) * 2, ' ');
+  const uint8_t* input = src.data();
+
+  for (size_t i = 0; i < src.size(); i++)
+  {
+    output[2 * i] = hexMap[(input[i] & 0xF0) >> 4];
+    output[2 * i + 1] = hexMap[input[i] & 0x0F];
+  }
+
+  return output;
+}
+
+TEST_F(KeyVaultKeyClient, CreateKeyWithReleasePolicyOptions)
+{
+  auto const keyName = GetTestName();
+  auto const& client = GetClientForTest(keyName);
+
+  Azure::Security::KeyVault::Keys::CreateKeyOptions options;
+  options.KeyOperations.push_back(Azure::Security::KeyVault::Keys::KeyOperation::Sign);
+  options.KeyOperations.push_back(Azure::Security::KeyVault::Keys::KeyOperation::Verify);
+  options.ReleasePolicy = KeyReleasePolicy();
+  options.ReleasePolicy.Value().Immutable = false;
+  std::string dataStr = "{"
```

Comment:
Use string literal here instead of concatenating the string (it makes it somewhat easier to read in printf statements and the debugger)?

Reply:
I don't see how that helps https://en.cppreference.com/w/cpp/language/string_literal 
it's exactly the same as what i did, it would still concatenate strings, and i prefer to have a formatted json vs one long interminable line, that would then be cut into pieces by the formatter, thus again concatenating strings.

Reply:
fiiiine , done :) you got me at debug